dvc/pkg/config.py

```diff
@@ -0,0 +1,35 @@
+from dvc.config import ConfigError
+
+
+def get_pkg_settings(config, name):
+    settings = config.config.get(config.SECTION_PKG_FMT.format(name))
```

Comment:
SECTION_PKG_FMT is package specific. Should be probably defined here.

Reply:
It is part of the config format, same as all the other config sections and values. We use them to verify config format, so if we move them we would again probably get cyclic imports for no good reason.